uw-research-computing/connecting.md

```diff
@@ -14,7 +14,7 @@ throughput or high performance compute systems. If you haven\'t, see our
 2.  [Logging In](#login)
     -   [On the command line (Mac/Linux)](#login-ssh)
     -   [Using an SSH program (Windows/Mac)](#login-putty)
-    -   [Logging in Automatically](#ssh-keys)
+    -   [Re-using SSH connections](#reuse-ssh)
```

Comment:
```suggestion
    -   [Re-using SSH connections](#c-transferring-multiple-files)
```

Reply:
How are these generated? Should this actually be:

```suggestion
    -   [Re-using SSH connections](#c-re-using-ssh-connections)
```

Reply:
@jasoncpatton I think you're right. everything is lowercase and all spaces/punctuation replaced with a dash.

Reply:
Ok, I'll commit these and we'll fix later if they're wrong.